main.cc

```diff
@@ -1917,6 +1918,12 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             // just ignore it and exit normally
             _exit(0);
             return 0;
+          } catch (const abort_requested_exception&) {
```

Comment:
It looks like indentation is broken throughout this code.

Reply:
unless you accept 2-characters wide indentation.
in any case, this is orthogonal, and sent separately as https://github.com/scylladb/scylladb/pull/15443

Reply:
Yes, it is pre-existing issue.